### DISCUSSION TITLE
No flat field calibration image of subset found

### DISCUSSION THREAD

**Discussion by Jefi4084** [iraf-community/iraf#148] [Q&A]
I am trying to apply my flat images to my science images with ccdproc *.fits, but when I run this command, I receive this error. The error does not specify what subset is wrong. I would appreciate your help.

**Comment by olebole** [comment]
Maybe you try to run it with each subset independently and check this out.

**Comment by Jefi4084** [comment]
I am sorry did you attach something to 'check this out' ? I cannot see it. I am still an IRAF novice, how would I run ccdproc *.fits with each subset independently? And to be clear I already combined my flats into median flat images, where I have FlatR.fits, FlatI.fits, FlatV.fits, and FlatB.fits, so ccdproc is where it applies this individual flat images to the science images according to their respective filters.

**Comment by olebole** [comment]
No, I mean: you could try out checking each subset individually.
However: I would really not recommend to start with IRAF these days anymore. IRAF use is strongly decreasing, and the number of people who could help you are fading away quickly. IRAF is definitively a dead end.
Instead, I would recommend to go for the Astropy universe; there [is already a ccdproc package available](https://ccdproc.readthedocs.io/).

**Comment by olebole** [comment]
Since this is probably not a bug in IRAF; I am going to close this. Please re-open if you feel different.
For help, I would recommend to use [iraf.net](https://iraf.net), if it is still alive.
